src/autocomplete/EmojiProvider.tsx

```diff
@@ -38,21 +37,19 @@ const EMOJI_REGEX = new RegExp('(' + EMOTICON_REGEX.source + '|(?:^|\\s):[+-\\w]
 
 interface IEmojiShort {
     emoji: IEmoji;
-    shortname: string;
     _orderBy: number;
 }
 
-const EMOJI_SHORTNAMES: IEmojiShort[] = EMOJI.sort((a, b) => {
+const EMOJI_SHORTCODES: IEmojiShort[] = EMOJI.sort((a, b) => {
     if (a.group === b.group) {
         return a.order - b.order;
     }
     return a.group - b.group;
 }).map((emoji, index) => ({
     emoji,
-    shortname: `:${emoji.shortcodes[0]}:`,
     // Include the index so that we can preserve the original order
     _orderBy: index,
-}));
+})).filter(o => o.emoji.shortcodes[0]);
```

Comment:
I think this filter isn't good, see below.

Reply:
Ah right, is this because some of the new emojis don't have a shortcode and thus we have no helpful thing to render for them? e.g

![image](https://user-images.githubusercontent.com/2403652/126140913-a766973a-67f1-4219-b97e-3a9c84db5cb0.png)


which is now not accessible via the Emoji autocomplete